extensions-contrib/compressed-bigdecimal/src/main/java/org/apache/druid/compressedbigdecimal/CompressedBigDecimal.java

```diff
@@ -274,13 +276,74 @@ protected static <S> int signumInternal(int size, S rhs, ToIntBiFunction<S, Inte
   /* (non-Javadoc)
    * @see java.lang.Comparable#compareTo(java.lang.Object)
    */
+
```

Comment:
accidental change? And I think the javadoc for this method is not helpful.

Reply:
yea, i'm not sure what happened here. there are a lot of these javadoc comments from the original PR that was merged :/

i'll  remove it